addons/point_of_sale/static/src/app/generic_components/product_card/product_card.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="point_of_sale.ProductCard">
+        <article tabindex="0" 
+            t-attf-class="product position-relative btn btn-light d-flex flex-column align-items-stretch p-0 m-0 text-start cursor-pointer overflow-hidden transition-base" 
+            t-on-keypress="(event) => event.which === 32 ? props.onClick() : ()=>{}" 
+            t-on-click="props.onClick" 
```

Comment:
is this not a problem if props.onClick is undefined?

Reply:
indeed. i made this change ```js
    static defaultProps = {
        onClick: () => {},
    };```